tests/comicarchive_test.py

```diff
@@ -16,7 +16,7 @@ def test_getPageNameList():
     pageNameList = c.get_page_name_list()
 
     assert pageNameList == [
-        "!cover.jpg",
+        "!cover.jpg",  # Depending on locale punctuation or numbers might come first (Linux)
```

Comment:
I'm having trouble getting pyicu to install, tried a few things. It's also why a couple of other tests fail for me locally but pass on here I expect. Anyway, it was more information for me to remember. :)

Reply:
Yes it's annoying to make it work, but it means that ComicTagger doesn't need to decide how things are sorted, it will be sorted almost exactly how it sorts in the filebrowser.